test/test_client.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+
+from conftest import AUTH_URL, AUTH_VERSION, auth_users  # @UnusedImport
```

Comment:
why import something if it is not used?

Reply:
It's used in the last 2 tests. PyDev doesn't realize it's a fixture since it just looks like an argument to a method

Reply:
Huh, it does not. Well that's neat, I've been doing that for years now. I wonder how I got that in my head.

Anyway, removed

Reply:
maybe I started putting the import there because otherwise it's not clear where the fixture comes from to the uninitiated